core/input/input_event.cpp

```diff
@@ -444,9 +444,9 @@ String InputEventKey::as_text() const {
 	} else if (keycode != Key::NONE) {
 		kc = keycode_get_string(keycode);
 	} else if (physical_keycode != Key::NONE) {
-		kc = keycode_get_string(physical_keycode) + " (" + RTR("Physical") + ")";
+		kc = keycode_get_string(physical_keycode) + " (" + TTR("Physical") + ")";
```

Comment:
These public API changes are technically compat-breaking and needs to be documented.

Reply:
Hmm, I think those should stay as `RTR()`, actually.